src/index.html

```diff
@@ -6,9 +6,63 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="#1" class="header__homelink">
```

Comment:
![image](https://github.com/mate-academy/layout_moyo-header/assets/96138503/f9c06afd-fa5f-4340-97ee-032390d7b228)
```suggestion
        <a href="#home" class="header__homelink">
```

Reply:
Pay attention to comments.

Reply:
Yesterday there was this comment, initially, I had "index.html" set up, and I thought it was the home page. I changed it to "#home," hoping it's correct.